mod.yaml

```diff
@@ -3,12 +3,12 @@
 Metadata:
 	Title: Over Powered Mod
 	Description: Crazy modifications to Red Alert by Boolbada
-	Version: 007-Playtest-20170304
+	Version: {DEV_VERSION}
 	Author: Boolbada
 
 RequiresMods:
-	#modchooser: playtest-20170304
 	modchooser: {DEV_VERSION}
+	as: {DEV_VERSION}
 
```

Comment:
I added that comment line again. This PR introduces a dependency on AS so there is no point in removing it's dependency line.